CheckFilelist.py

```diff
@@ -136,6 +136,14 @@ def ghostfile(pkg, f):
             '/usr/lib/perl5/site_perl/*',
         ],
     },
+    {
+        'error': 'suse-filelist-forbidden-pyton-test-dir',
+        'details': '''python package installs testsuite to the sitelib,
+                        which causes conflict and is not allowed in SUSE.''',
+        'bad': [
+            '/usr/lib*/python*/site-packages/test',
```

Comment:
Missing /* at the end? Or why would the directory alone be a problem?

Reply:
Folder itself is an issue.